build/config/ESRPSigning_Terminal.json

```diff
@@ -0,0 +1,117 @@
+{
+    "Version": "1.0.0",
+    "UseMinimatch": false,
+    "SignBatches": [
+        {
+            "MatchedPath": [
+                // Namespaced DLLs
+                "Microsoft.Terminal.*.dll",
+                "Microsoft.Terminal.*.winmd",
+
+                // ConPTY and DefTerm
+                "OpenConsole.exe",
+                "OpenConsoleProxy.dll",
+
+                // VCRT Forwarders
+                "*_app.dll",
+
+                // Legacy DLLs with old names
+                "TerminalApp.dll",
+                "TerminalApp.winmd",
+                "TerminalConnection.dll",
+                "TerminalThemeHelpers.dll",
+                "WindowsTerminalShellExt.dll",
+
+                // The rest
+                "TerminalAzBridge.exe",
+                "wt.exe",
+                "WindowsTerminal.exe",
+                "elevate-shim.exe"
```

Comment:
Couldn't we make this something like a catch-all clause with `*.exe`, `*.dll`, and `*.winmd` as the MatchedPaths by putting it after the 3rd party object/section below? That way we won't have to keep track of all these binaries in the future.

Reply:
That's a fair question. I don't know if it excludes files that were included in a previous batch. I'd rather not add a rule that signs all EXEs and DLLs for now, but if this becomes burdensome we can revisit it. Thanks :)